cmd/kubelet/app/options/container_runtime.go

```diff
@@ -33,7 +33,7 @@ const (
 
 var (
 	defaultPodSandboxImage = defaultPodSandboxImageName +
-		":" + defaultPodSandboxImageVersion
+		"-amd64:" + defaultPodSandboxImageVersion
```

Comment:
/hold

@wgliang 
i don't think that hard coding a platform suffix is the correct solution for this.

@chuckha please, close the PR or request changes.